Docs/NugetREADME.md

```diff
@@ -0,0 +1,50 @@
+# OPC UA .NET Standard stack documentation
+
+## Overview
+
+The OPC UA .NET Standard stack enables you to build multi-platform OPC UA Applications with rich functionality including:
+ - Server
+ - Client
+ - PubSub
+
+
+### For more information and license terms, see the projects official [Website](http://opcfoundation.github.io/UA-.NETStandard).
+
+## Getting started
+
+The reference [Client](https://github.com/OPCFoundation/UA-.NETStandard/tree/master/Applications/ConsoleReferenceClient) & [Server](https://github.com/OPCFoundation/UA-.NETStandard/tree/master/Applications/ReferenceServer) projects provide a starting point in implementing your own application.
+
+Simple Client:
+```C#
+// Define the UA Client application
+ApplicationInstance application = new ApplicationInstance();
+application.ApplicationName = "My OPC UA Client";
+application.ApplicationType = ApplicationType.Client;    
+
+// load the application configuration.
```

Comment:
Would be cool if this were a sample that really compiles, with the fluent ApplicationConfiguration... 
But the UAClient is not a shipped lib :-(

Reply:
Thanks for bringing that up. I am aware of that, i will Look into Long Term solution, short Term we will need to create the readme without Sample Code to stay simple